packages/core-web/src/index.js

```diff
@@ -75,31 +88,36 @@ function detectAndApplyFixedHeaderStyles() {
     if (isEndOfPage) { return; }
 
     if (currentOffset > lastOffset) {
-      const headerEnd = headerSelector.height() + headerSelector[0].getBoundingClientRect().top;
-      const isBeforeHeader = currentOffset < headerEnd;
+      /*
+       1) Bounding box is calculated as if 'position: fixed' when sticky is "activated".
+       Revert the position to 'static' temporarily to avoid this.
+
+       Seems to be harmless UX wise, even on extremely slow devices.
```

Comment:
The change here is pretty much as explained in the comment, although I would like to get some help with testing this ^ in particular on more devices. (whether the header ever "flickers" right before it is being hidden)

Reply:
Tested and works fine on my end 👍